colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -0,0 +1,370 @@
+      },
+      "source": [
+        "# Use an Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jZC69sSMdX4j"
+      },
+      "source": [
+        "When you want to use a specific version of an Artifact in a downstream task, you can specify the specific version you would like to use via either `v0`, `v1`, `v2` and so on, or via specific aliases you may have added. The `latest` alias always refers to the most recent version of the Artifact logged.\n",
+        "\n",
+        "The proceeding code snippet specifies an artifact called `my_first_artifact` with the alias `latest`:\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "7Ix_ciolJICe"
+      },
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basic\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\")\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "u60Orwg5sYuI"
+      },
+      "source": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Create a new Artifact version"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "When you need add a new file, you can use the [`artifact.add_file`](https://docs.wandb.ai/ref/python/artifact#add_file) method."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(\"my_first_artifact:latest\") # selects the artifact you're adding the file to\n",
+        "artifact.add_file(local_path=\"/content/sample_data/california_housing_test.csv\", name=\"new_file\")\n",
+        "run.log_artifact(artifact)\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "This adds a new .csv file called `new_file` to the `my_first_artifact` Artifact. \n",
+        "\n",
+        "If you edit a file, you'll need to go through a similar process:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# sorts the .csv file\n",
+        "import pandas\n",
+        "csvData = pandas.read_csv(\"/content/sample_data/california_housing_test.csv\") \n",
+        "csvData.sort_values(csvData.columns[6],  \n",
+        "                    axis=0, \n",
+        "                    inplace=True)\n",
+        "csvData.to_csv(\"/content/sample_data/california_housing_test.csv\") # overwrites file with the sorted data\n",
+        "# adds the new file to the artifact\n",
+        "run = wandb.init(project=\"artifact-basics\")\n",
```

Comment:
I think would be better here to init the run at the start of the code block.

Reply:
- maybe split in 2 cells. 
- `csv_data` instead of `csvData`.